tgui/packages/tgui/interfaces/ChemHeater.js

```diff
@@ -271,21 +271,21 @@ export const ChemHeater = (props, context) => {
                         </Box>
                       )}
                     </TableCell>
-                  </TableRow> 
+                  </TableRow>
```

Comment:
Change to `<Table.Row>`, same goes for `<Table.Cell>`, remove imports of these components.

Reply:
Done